src/layers/drop.jl

```diff
@@ -126,36 +121,39 @@ function Base.show(io::IO, d::DropBlock)
     return print(io, ")")
 end
 
-# TODO look into "row" mode for stochastic depth
 """
-    DropPath(p; [rng = rng_from_array(x)])
```

Comment:
To make this non/less breaking, adding a deprecation for `DropPath` would work.

Reply:
But the Layers API is unexported, so does this still count as a breaking change?

Reply:
It really should be exported in 0.8, but for now it isn't so I suppose it might work out in our favour 😄

Reply:
I assumed it was why you added the label, but that is true. Built docs list it as "public", but I think that's a false positive and JuliaHub tells me no registered packages are using it.

Reply:
Oh no, the breaking change is that `EfficientNet` literally returns a different model 😅